core-bundle/tests/Functional/app/config/security.yaml

```diff
@@ -1,14 +1,16 @@
 security:
+    enable_authenticator_manager: true
```

Comment:
I thought this is enabled in Sf 6 by default/cannot be disabled?

Reply:
Correct, is not needed anymore. Removed in b2814d4562.

Reply:
I'll have to revert: https://github.com/symfony/security-bundle/blob/6.1/DependencyInjection/SecurityExtension.php#L97

Reply:
Reverted in 099c9a4b69.

Reply:
Forget all about it - it was buggy in 6.0.0 and 6.0.1. Need to require 6.0.2 (fixed in 01a42c5228).